### ISSUE TITLE
Recreações culturais e esportivas not updating for Kajetan

### ISSUE THREAD

**Issue by zross** [worldbank/simvat#101]
@ktrzcinskiwb I'm pulling this out of issue #100. Can you please give me details on this? I see it happening in your video but it does not happen for me locally or in the deployed app.

One thing that I see that might be hard to address is:

1. Make all of that category 0
2. Make Recreações culturais e esportivas 20 (or whatever)
3. Try to make all of that category 0 again -- if you do it quickly, the 20 in Recreações culturais e esportivas will not change

This is because Shiny is designed not to trigger updates if an input does not change. So since you're putting 0 in an input that is already 0 it does not update anything.

Does the issue you show me in the video happen when you first arrive at the app or did you make changes?

**Comment by zross** [comment]
@ktrzcinskiwb you wrote 

> IMPORTANT: I just noted that we have a regular category called "Recreações culturais e esportivas" and a big macro-category with the same name. Could this be creating the conflict?

Yes, this can cause a problem. All interactive elements in need a unique ID. I am using a cleaned version of the names and for both of those I am using `recreacoes_culturais_e_esportivas` which is supposed to be unique. I'm not sure what problems this is creating, but definitely some problems. I can do something to adjust this.

**Comment by ktrzcinskiwb** [comment]
Should we just change one of the names? If that is a quick fix on your side we can adjust.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by zross** [comment]
No, I've made a change that should work. I'm going to deploy right now. (But you should take note of what I spelled out above, that's fairly important but probably very hard for me to change).

**Comment by ktrzcinskiwb** [comment]
Thank you Zev, will test it as soon as I can. Just to be clear, it won’t change but it will be clear that the cell hasn’t changed, ie the parameters the user sees are the ones in the calculation? I think we can live with that, my concern is that with the error we had yesterday I would change the parameter to 0 but it didn’t seem to appear in the calculations.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by zross** [comment]
You have it correct: What they see in the categories (not the macro category) will be what is used.
